mixins/eirinix/templates/dns-aliases/service.yaml

```
@@ -0,0 +1,20 @@
+{{- $_ := include "_config.load" $ }}
+{{- if .Values.features.eirini.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: eirini-dns-aliases
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- list . "eirini-dns-aliases" | include "component.labels" | nindent 4 }}
+spec:
+  type: ClusterIP
+  selector:
+    {{- list . "eirini-dns-aliases" | include "component.selector" | nindent 4 }}
+  ports:
+  - protocol: TCP
+    name: webhook
+    port: 443
```

If you are using latest EiriniX, you can just

```suggestion
    port: 8443
```

---

for reference: https://github.com/cloudfoundry-incubator/kubecf/blob/master/mixins/eirinix/templates/loggregator-bridge.yaml#L106